progressReport/app.py

```diff
@@ -126,6 +155,51 @@ def assign_node_levels(node, student_levels_count, class_levels_count):
                            course_data=course_nodes)
 
 
+@app.route('/', methods=["POST"])
+def generate_cm_from_post_parameters():
+    request_as_json = request.get_json()
+    validate_mastery_learning_post_request(request_as_json)
+    school_name = request_as_json.get("school", "Berkeley")
+    course_name = request_as_json.get("class", "CS10")
```

Comment:
Thought I would designate the default school and class values as `"Berkeley"` and `"CS10"` respectively if those parameters are not included in the request, because William’s code used those defaults. Good point, though; I should make the default values constants, so that they can be set more easily.